src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -85,6 +85,21 @@ var ReactPropTypes = {
   shape: createShapeTypeChecker,
 };
 
+/**
+ * inlined Object.is polyfill to avoid requiring consumers ship their own
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is
+ */
+function is(x, y) {
```

Comment:
This should move to a separate file at https://github.com/facebook/react/tree/master/src/shared/stubs